### ISSUE TITLE
Ibutsu Jenkins Pipeline Heatmap only consider sequential run results

### ISSUE THREAD

**Issue by ntkathole** [ibutsu/ibutsu-server#292]
We are seeing Ibutsu Jenkins Pipeline Heatmap widget only consider 1 run if there are multiple run in same jenkins job for same plugin.

**Problem statement:**

We run some tests from test suite in sequential and some tests in parallel. So it creates 2 different run in ibutsu. 

When we create Jenkins Pipeline Heatmap widget for such jobs, it only considers sequential run results and ignore parallel run results.

**Comment by john-dupuy** [comment]
Hey @ntkathole - this is a duplicate of #109 - unfortunately this is a very specific widget and will need some hacks to support this. 

I will close this one - feel free on continue discussion on #109
